README.md

```diff
@@ -150,125 +144,6 @@ In addition to the standard fields for all articles, check the specific fields f
 
 - **troubleshootingtags**: Troubleshooting tag information.
```

Comment:
explain how they should be added, e.g. with `"`? separated by comma? capitalized or not?

Reply:
Fixed!